src/asciidoc/docs/develop/publishLocal.adoc

```diff
@@ -0,0 +1,240 @@
+To see which files have changed, use the command `git status`.
+
+NOTE: The `.gitignore` file at the root of your project lists the files that Git does not track for changes.
+You might not see the file if your operating system is configured to hide files whose names begin with a dot.
+In particular, this file tells Git to ignore any changes to files in the `node_modules` folders, which were created when you ran `npm install` in the `frontend` and `backend` folders.
+
+When you are ready to test your changes on the Koji server, use Git to `commit` your changes locally.
+
+. Open a terminal window and go to the root directory of your project (for example, `MyKojiTemplate`).
+. Run the following Git commands.
++
+[source,bash]
+----
+$ git add . <1>
+$ git commit -m "your comments" <2>
+----
+<1> Processes all the documents that you have changed.
+<2> Creates an updated version of your project.
++
+[NOTE]
+====
+Be sure to replace `your comments` with a brief description of the changes that you have made, so that you will be able to remember what features you were working on.
+
+For a more complete description of the `git add` and `git commit` commands, see https://www.atlassian.com/git/tutorials/saving-changes[this tutorial].
+====
+
+=== Pushing changes to the Koji repository
+
+After the commit process is successfully completed, you use Git to upload the committed changes to the repository on the Koji servers.
+
+. Open a terminal window and go to the root directory of your project.
+. Run the following command.
++
+[source,bash]
+git push origin master
+. At the username prompt, enter your username on Koji and press *Enter*.
+. At the password prompt, enter the access key that you <<use-git#_obtaining_an_access_key, generated in a previous tutorial>> and press *Enter*.
++
+Your terminal will look something like this:
++
+[source,bash]
+----
+$ git push origin master <1>
+Counting objects: 21, done.
+Delta compression using up to 4 threads.
+Compressing objects: 100% (21/21), done.
+Writing objects: 100% (21/21), 84.53 KiB | 0 bytes/s, done.
+Total 21 (delta 15), reused 0 (delta 0)
+Username for 'https://projects.koji-cdn.com': KojiCoder
+Password for 'https://KojiCoder@projects.koji-cdn.com': <2>
+To https://projects.koji-cdn.com/a70f8329-e89e-48b0-8d85-7658c1542b9f.git
+  a88036c..ea6bda1  master -> master
+----
+<1> Use Git to upload your new version to the Koji repository
+<2> Paste your 32-character access key.
+
+[NOTE]
+Pushing your changes to origin automatically publishes a new version of your app.
+
+=== Testing your published app
+
+Now that you've published your app, you will want to make sure that it works the same on the Koji server as it does locally.
+
+1. Go to the https://withkoji.com/[Koji home page] and sign in to your account.
+2. Click your profile button and go to *Apps > Developer Portal*.
+3. Select your app from *All Projects*.
+4. Click *View Published Koji*.
```

Comment:
From the project details, they have a few options that we might want to call out (rather than just View Published Koji). Specifically, it would be nice to add something here about how you can run the debugger for something you are developing locally. E.g., overriding the environment settings. You can just mention it and then link to the debugging page. Also, might be nice to mention that you can verify the status of the build/deploy here.